packages/integration-tests/projects/suite-web/tests/wallet/custom-blockbook-discovery.test.ts

```diff
@@ -0,0 +1,56 @@
+// @group:suite
+// @retry=2
+
+describe('Custom-blockbook-discovery', () => {
+    beforeEach(() => {
+        cy.task('startEmu', { wipe: true });
+        cy.task('setupEmu', {
+            needs_backup: true,
+            mnemonic: 'all all all all all all all all all all all all',
+        });
+        cy.task('startBridge');
+
+        cy.viewport(1080, 1440).resetDb();
+        cy.prefixedVisit('/settings/coins');
+        cy.getTestElement('@suite/menu/settings').click();
```

Comment:
i guess you want to "pass thru onborading" before each tests
```
cy.prefixedVisit('/settings/coins');
cy.passThroughInitialRun();
```

then you dont have to click on menu to get in to settings (you should be there already)

you can get inspiration from existing tests:
https://github.com/trezor/trezor-suite/blob/develop/packages/integration-tests/projects/suite-web/tests/wallet/discovery.test.ts

or

https://github.com/trezor/trezor-suite/blob/develop/packages/integration-tests/projects/suite-web/tests/settings/coins.test.ts

Reply:
thank you for the tip, will do!